op-devstack/devtest/testing.go

```diff
@@ -240,7 +239,15 @@ func SerialT(t *testing.T) T {
 	t.Cleanup(func() {
 		span.End()
 	})
-	logger := NewLogger(ctx, t, log.LevelInfo).WithContext(ctx)
+	logLevel := os.Getenv("TEST_LOG_LEVEL")
+	if logLevel == "" {
+		logLevel = "info" // default to info if not set
+	}
+	level, err := log.LevelFromString(logLevel)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Can we do a test-fail instead of panic here? I don't think outside env var values should ever result in panics.